recipes-extended/xen/xen-tools_git.bbappend

```diff
@@ -0,0 +1,12 @@
+SRCREV = "${AUTOREV}"
+XEN_REL = "4.12"
+LIC_FILES_CHKSUM = "file://COPYING;md5=bbb4b1bdc2c3b6743da3c39d03249095"
+
+# OpenXT's Xen recipes share a common patchqueue so reset SRC_URI
+SRC_URI = "git://xenbits.xen.org/xen.git;branch=${XEN_BRANCH}"
+
+require xen-common.inc
+require ${@bb.utils.contains('DISTRO_FEATURES', 'blktap2', 'xen-tools-blktap2.inc', 'xen-tools-blktap3.inc', d)}
+require xen-tools-openxt.inc
+
+DEFAULT_PREFERENCE = "1"
```

Comment:
Not sure I understand why `DEFAULT_PREFERENCE` is set to `1` here, it shouldn't change anything afaict.

Reply:
Since Xen 4.14 is now released, I'm about to swap the `DEFAULT_PREFERENCE` value in the upstream meta-virtualization Xen recipes, which will make the `xen_git.bb` and `xen-tools_git.bb` recipes non-default.

It'll only matter if OpenXT's Xen major version happens to match the major stable version in meta-virt, which at some point it hopefully will. The aim is to avoid any surprise accidental use of the non-"_git.bb" Xen recipes from meta-virt.

Reply:
Makes sense.